plugin/plugin.go

```diff
@@ -33,6 +33,9 @@ type Args struct {
 
 // Exec executes the plugin.
 func Exec(ctx context.Context, args Args) error {
+	// for jfrogv2 cli, to hide unwanted messages
+	os.Setenv("CI", "true");
```

Comment:
`CI=true` is automatically passed into every plugin by Drone.  Do we want to automatically set this value in the Go code as well or should it be set by CIE?  If we do want to set in Go code, you could add this environment variable to the Dockerfile, or you could set this value in `os.Exec.Env` instead of setting this at the process level.

Reply:
@bradrydzewski Addressed your comment, have put the ENV to Dockerfile.